COORDINATION.md

```diff
@@ -87,6 +125,7 @@ meeting the timing constraints around enumeration.  Dynamic adding and
 deleting devices on the PCI bus via hot plug requires BIOS configuration.
 
 ## 3: Out-band via platform BMC
+***SB> How about multi Host support?***
```

Comment:
To some degree I see an xPU as multihost.  The processor on the xPU and host processor are sharing a network device.  The key thing is power domains and if the multiple hosts need to be coordinated.  It may be possible to have multiple BMCs on I2C in a multi-master configuration.